api_specifications/case-api-0.1.0.yaml

```
@@ -1340,3 +1396,106 @@ components:
+      properties:
+        requestInfo:
+          $ref: 'https://raw.githubusercontent.com/egovernments/DIGIT-OSS/master/core-services/docs/common-contract_v1-1.yml#/components/schemas/RequestInfo'
+        ProcessInfo:
+          type: object
+          properties:
+            caseId:
+              type: string
+              description: "uuid of the case"
+            action:
+              type: string
+              enum: [ "APPROVE", "REJECT" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)